.bazelrc

```diff
@@ -10,6 +10,7 @@ build:production --config=lsan --compilation_mode=dbg --strip=never --copt=-O3
 # C/C++ CONFIGS
 build --cxxopt=-std=c++14
 build --experimental_strict_action_env
+build --compilation_mode=dbg
```

Comment:
nitpicking: comment for line 8: is `--compilation_mode=dbg` now redundant?

Reply:
yeah true, will clean this up